src/frontend/settingsdialogs/bttexttospeechsettings.cpp

```diff
@@ -0,0 +1,148 @@
+/*********
+*
+* In the name of the Father, and of the Son, and of the Holy Spirit.
+*
+* This file is part of BibleTime's source code, https://bibletime.info/
+*
+* Copyright 2025 by the BibleTime developers.
+* The BibleTime source code is licensed under the GNU General Public License
+* version 2.0.
+*
+**********/
+
+#ifdef BUILD_TEXT_TO_SPEECH
+
+#include "bttexttospeechsettings.h"
+
+#include <QComboBox>
+#include <QFormLayout>
+#include <QLabel>
+#include <QLocale>
+#include <QVoice>
+#include "../../backend/config/btconfig.h"
+#include "../../util/btconnect.h"
+#include "../../util/cresmgr.h"
+#include "../bibletime.h"
+#include "cconfigurationdialog.h"
+
+namespace {
+
+QString formatLocale(const QLocale& locale) {
+    return QObject::tr("%1 (%2)", "Locale language and territory")
+        .arg(QLocale::languageToString(locale.language()),
+             QLocale::territoryToString(locale.territory()));
+}
+
+}
```

Comment:
I suggest adding a `// anonymous namespace` comment to the closing brace.

Reply:
Done